core/templates/dev/head/app.js

```diff
@@ -517,7 +517,9 @@ oppia.factory('rteHelperService', [
       },
       createToolbarIcon: function(componentDefn) {
         var el = $('<img/>');
-        el.attr('src', componentDefn.iconDataUrl);
+        el.attr('src',
```

Comment:
Break after '(' if contents of (...) span multiple lines.

Reply:
Done.